src/legacy/ui/public/agg_types/controls/string.tsx

```diff
@@ -17,12 +17,19 @@
  * under the License.
  */
 
+import { isFunction } from 'lodash';
 import React from 'react';
 
 import { EuiFieldText, EuiFormRow } from '@elastic/eui';
 import { AggParamEditorProps } from '../../vis/editors/default';
 
 function StringParamEditor({ agg, aggParam, value, setValue }: AggParamEditorProps<string>) {
+  if (isFunction(aggParam.disabled) && aggParam.disabled(agg)) {
```

Comment:
Since we're using TS here, as long as `aggPram.disabled` is truthy it will always be a function. So I would suggest we just check for `aggParam.disabled && aggParam.disabled(agg)` here. That will not require us to import anything from lodash here.